include/dds/sub/SubscriberListener.hpp

```diff
@@ -98,7 +98,7 @@ class NoOpSubscriberListener;
  * @see @ref DCPS_Modules_Infrastructure_Listener "Listener information"
  */
 // TODO Uncomment when PSM listeners are implemented.
-//class OMG_DDS_API SubscriberListener : public virtual AnyDataReaderListener
+//class OMG_DDS_API SubscriberListener : public virtual AnySubscriberListener
```

Comment:
I think it was correct, as the standard said that SubscriberListener inherits from DataReaderListener